frontend/src/app/general/header/header.component.html

```diff
@@ -20,8 +20,11 @@
     <div class="ml-5 basis-1/3 select-none text-2xl text-primary">
       Capella Collaboration Manager
     </div>
-    <div class="flex basis-1/3 gap-2">
-      @for (item of navBarService.navBarItems; track item.name) {
+    <div
+      class="flex basis-1/3 gap-2"
+      *ngIf="navBarService.navbarItems$ | async"
```

Comment:
Don't use `ngIf` or `ngFor`. That's the old Angular syntax, there is a new control flow syntax with `@if` and `@for`: https://v17.angular.io/guide/control_flow